src/plugins/vis_type_table/public/to_ast.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EsaggsExpressionFunctionDefinition } from '../../data/common/search/expressions';
+import { buildExpression, buildExpressionFunction } from '../../expressions/public';
+import { getVisSchemas, Vis, BuildPipelineParams } from '../../visualizations/public';
+import { TableExpressionFunctionDefinition } from './table_vis_fn';
+import { TableVisConfig, TableVisParams } from './types';
+
+const buildTableVisConfig = (
+  schemas: ReturnType<typeof getVisSchemas>,
+  visParams: TableVisParams
+) => {
+  const visConfig = {} as any;
```

Comment:
I know that this `any` existed before but I was wondering if it can be easily typed.

Reply:
I'll update this in a follow up pr here https://github.com/elastic/kibana/pull/70801 , if you don't mind.

Reply:
You are the best!